components/Layout/Navigation/Nav.tsx

```diff
@@ -13,13 +16,13 @@ export default function Nav({ toggle }) {
   )
 }
 
-// todo: add styles
 const NavMenu = styled.nav`
   margin: 0;
   padding: 0;
   display: flex;
   justify-self: end;
   font-size: 2rem;
+
   a,
   button {
```

Comment:
You're using styled components. So use the styled components way; abstract these to it's own styled component

Reply:
wouldnt that lead to repetition? I want them to have the same style.

Reply:
I always encourage to use the DRY principle, that's true, but it shouldn't never be at the expense of good clean code (which also implies using the best practices and following the way it's intended).

In order to avoid repetition you could, e.g,  do the following:

```javascript
const ButtonLikeThingyStyled = styled.button` <-- notice button
   background-color: red;
`;
```
The cool thing about Styled components is that you can force it to be something else, even though you set it to be a button like so:
```javascript
// this will output a button
<ButtonLikeThingyStyled>
   Hey
</ButtonLikeThingyStyled>

// this will output an anchor
<ButtonLikeThingyStyled as="a">
 Heyya
</ButtonLikeThingyStyled>
```

Styled components accepts the prop "`as`" and will use that over what you've set in the variable